roles/lib_openshift/src/class/oc_route.py

```diff
@@ -55,7 +55,13 @@ def create(self):
 
     def update(self):
         '''update the object'''
-        # need to update the tls information and the service name
+        # when the host attribute is being updated, we need to delete and recreate
```

Comment:
Arrgh, I verified this is the case against the api directly as well.

What about something like this instead:
```
force = False
if self.config.host != self.route.host:
    force = True

return self._replace_content(self.kind, self.config.name, self.config.data, force=force)
```